amida-tech/indaba-client
# INBA-449 / Tiny change that properly perpetuates the state.

#### What does this PR do?
There was a very tiny bug where when a user selects a different notify level (an admin shouldn't be allowed to choose if he's active or not-- in fact, the inactive checkbox should probably go away as this is for the _current_ user and not another user's profile modal), an error action is called. 

The call to the endpoint works fine. The problem was just that nothing was being returned and the state wasn't being corrected to the proper values. Easy enough fix that we just call for a fresh profile afterwards. I'd have preferred to just modify the requestBody, but it's less code to just to call and be given fresh data. 

#### Related JIRA tickets:

#### How should this be manually tested?
Log in as an admin. Click on the "Man" icon on the right. Choose a different notification level. Save. Confirm no errors in the console log. Go to another page. Then navigate back to the Profile page. Confirm the notification level persists.

Do the same for user. 

#### Background/Context

#### Screenshots (if appropriate):
